src/main/java/net/pms/io/WindowsNamedPipe.java

```diff
@@ -57,7 +57,7 @@
 	private static final int BUFSIZE = 500000;
 
 	public interface Kernel32 extends StdCallLibrary {
-		Kernel32 INSTANCE = (Kernel32) Native.loadLibrary("kernel32",
+		Kernel32 INSTANCE = (Kernel32) Native.load("kernel32",
```

Comment:
This change (and the others) should be decoupled from this branch

Reply:
@ @SubJunk why? This is the Java 14 implementation and this is the actual version we use.

Reply:
I agree with the change, I just want it to be in a separate branch so it can merge faster and make this one easier to review :)

Reply:
@SubJunk just merge the #2231 and it will solve this.